GateWatcher/aioniq_ecs/ingest/parser.yml

```
@@ -0,0 +1,768 @@
+          gatewatcher.matched_event.file.magic: '{{json_load.message.matched_event.file.magic}}'
+          gatewatcher.matched_event.file.state: '{{json_load.message.matched_event.file.state}}'
+          gatewatcher.matched_event.file.tx_id: '{{json_load.message.matched_event.file.tx_id}}'
+          gatewatcher.matched_event.file.stored: '{{json_load.message.matched_event.file.stored}}'
+          gatewatcher.matched_event.file.hash.md5: '{{json_load.message.matched_event.file.hash.md5}}'
+          gatewatcher.matched_event.file.hash.sha256: '{{json_load.message.matched_event.file.hash.sha256}}'
+        filter: ''
+        name: set
+  ransomware_detect:
+    actions:
+      - set:
+          gatewatcher.ransomware.session_score: '{{json_load.message.ransomware.session_score}}'
+          gatewatcher.ransomware.alert_threshold: '{{json_load.message.ransomware.alert_threshold}}'
+          gatewatcher.ransomware.malicious_behavior_confidence: '{{json_load.message.ransomware.malicious_behavior_confidence}}'
+        filter: ''
+        name: set
+  dga_detect:
+    actions:
+      - set:
+          gatewatcher.dga.top_DGA: '{{json_load.message.dga.top_DGA}}'
+          gatewatcher.dga.dga_count: '{{json_load.message.dga.dga_count}}'
+          gatewatcher.dga.dga_ratio: '{{json_load.message.dga.dga_ratio}}'
+          gatewatcher.dga.nx_domain_count: '{{json_load.message.dga.nx_domain_count}}'
+          gatewatcher.dga.malware_behavior_confidence: '{{json_load.message.dga.malware_behavior_confidence}}'
+        filter: ''
+        name: set
+  network_behavior_analytics:
+    actions:
+      - set:
+          gatewatcher.nba.gid: '{{json_load.message.nba.gid}}'
+          gatewatcher.nba.rev: '{{json_load.message.nba.rev}}'
+          gatewatcher.nba.action: '{{json_load.message.nba.action}}'
+          gatewatcher.nba.packet: '{{json_load.message.nba.packet}}'
+          gatewatcher.nba.stream: '{{json_load.message.nba.stream}}'
+          gatewatcher.nba.payload: '{{json_load.message.nba.payload}}'
+          gatewatcher.nba.category: '{{json_load.message.nba.category}}'
+          gatewatcher.nba.signature: '{{json_load.message.nba.signature}}'
+          gatewatcher.nba.signature_id: '{{json_load.message.nba.signature_id}}'
+          gatewatcher.nba.payload_printable: '{{json_load.message.nba.payload_printable}}'
+          gatewatcher.nba.metadata.performance_impact: '{{json_load.message.nba.metadata.performance_impact}}'
+          gatewatcher.nba.metadata.signature_severity: '{{json_load.message.nba.metadata.signature_severity}}'
+        filter: ''
+        name: set
+  ioc:
+    actions:
+      - set:
+          gatewatcher.ioc.tlp: '{{json_load.message.ioc.tlp}}'
+          gatewatcher.ioc.tags: '{{json_load.message.ioc.tags}}'
+          gatewatcher.ioc.type: '{{json_load.message.ioc.type}}'
+          gatewatcher.ioc.value: '{{json_load.message.ioc.value}}'
```

Cannot ECS `threat.indicator.*` fields be used here?
```suggestion
          threat.indicator.marking.tlp: '{{json_load.message.ioc.tlp}}'
          gatewatcher.ioc.tags: '{{json_load.message.ioc.tags}}'
          threat.indicator.type: '{{json_load.message.ioc.type}}'
          threat.indicator.name: '{{json_load.message.ioc.value}}'
```
With a `translate` action for `threat.indicator.type` to map the expected values in the ECS specification for this field.